lighthouse-core/audits/byte-efficiency/uses-webp-images.js

```diff
@@ -21,10 +21,10 @@ class UsesWebPImages extends ByteEfficiencyAudit {
   static get meta() {
     return {
       name: 'uses-webp-images',
-      description: 'Serve images as WebP',
+      description: 'Serve images in superior formats',
```

Comment:
"Superior formats" seems a little judgy. Can we just say "next-gen formats" or "performant formats"

Reply:
'next-gen' was nixed, but performant SGTM

@paulirish?

Reply:
nevermind paul's good with "next-gen" afterall 👍